features/storage/csi_clone.feature

```diff
@@ -0,0 +1,67 @@
+Feature: CSI clone testing related feature
+  """
+  This test can only work on a Cluster which has CSI driver deployed, OSP has csi deploy by default from 4.7. 
+  When deploy a cluster which has no csi deployed, please use enable_tpnu_featureset: "yes" in LAUNCHER_VARS.
+  From 4.7, OSP CSI driver was installed by default. Its storage class name is standard-csi.
+  Use below command can temporarily add cs class(oc get sc, can find all deployed stroage class):
+  oc patch featuregate cluster -p '{"spec": {"featureSet": "TechPreviewNoUpgrade"}}' --type merge
+  """
+  
+  # @author jianl@redhat.com
+  # @case_id OCP-27615
+  @admin
+  Scenario: Clone a pvc on OSP in the same namespace is successfully
+    Given the master version >= "4.8"
+    Given I switch to cluster admin pseudo user
+    # Step 1
+    # From 4.7, OSP CSI driver was installed by default. The storage class name is standard-csi
```

Comment:
Let's remove the "step 1" as the we will only run it on the platform which have already installed the CSI driver, and all the info is added in your previous comments.
And restart the other steps from "step 1".

Reply:
@duanwei33 Sure

Reply:
@JianLi-RH Is it possible to add some word in the comments for the step, I guess only "step x" may be not so helpful.

Reply:
I just want to keep all steps separated clearly, it depends on Polarion test steps. No more meaning... @duanwei33